finance/proposal-calls/internal_template.md

```diff
@@ -0,0 +1,56 @@
+# Phase II funding proposals
+
+*This is a draft document for how the Interim Finance Committee envisions the process of distributing grant money (e.g. from the Moore grant) within the Astropy project. This is a general template and the finance committee will change the details for individual calls. In particular, actual dollar numbers and details of the grant and grant rules that limit the work that could be proposed will need to be filled in. This template is written with the current Moore grant in mind.*
```

Comment:
'dollar amounts' -> US centric wording, maybe generalize in case of non-US funding some day